.gitignore

```diff
@@ -46,3 +46,5 @@ build
 *.synctex.gz
 *.out
 *.log
+*.code-workspace
```

Comment:
Looks good to me; these two lines seem to contain the relevant part of what is recommended on: 

https://www.toptal.com/developers/gitignore/api/visualstudiocode

Just one minor comment: could you add a line that consists of a comment indicating that VS Code related part begins there (after L48, that is)? See L39 in the current `.gitignore` for a similar comment about LaTeX. Thanks!

Reply:
I merged this PR but I ended generating another one with https://www.toptal.com/developers/gitignore/ but I also added other patterns to ignore.